app/views/boxes/show.haml

```diff
@@ -30,17 +30,29 @@
       .col
         = render "barcode_card"
 
-        .barcode-after
+.row
+  .col.pe-3
+    %label Contents
+  .col
+    .row
+      .col.pe-3
+        .samples-count
+          .title
+            = @box.samples.count.to_s + ' ' + "samples" + (@box.blinded ? ' (blinded)' : '' )
```

Comment:
Let's interpolate:
```suggestion
            = "#{@box.samples.count} samples" + (@box.blinded ? ' (blinded)' : '')
```

Reply:
This was copypasted from the batches' samples screen, I changed it here but didn't modify it for the batches... should do it?

Reply:
Sure, let's :)